compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs

```diff
@@ -997,13 +1028,18 @@ convertImplements env mc tpl = NM.fromList <$>
       con <- convertInterfaceTyCon env handleIsNotInterface iface
       let mod = nameModule (getName iface)
 
-      -- TODO: Stub view, will extract later, https://github.com/digital-asset/daml/pull/14439
-      let view = ETmLam (ExprVarName "this", TCon con) (EBuiltin BEUnit)
-
       methods <- convertMethods $ MS.findWithDefault []
         (mod, qualObject con, tpl)
         (mcInterfaceMethodInstances mc)
 
+      view <- case MS.lookup (mod, qualObject con, tpl) (mcInterfaceViewInstances mc) of
```

Comment:
For templates, the equivalent check is done in [digital-asset/ghc - `RdrHsSyn.validateTemplate`](https://github.com/digital-asset/ghc/blob/da-master-8.8.1/compiler/parser/RdrHsSyn.hs#L3110-L3139) and in this file [`convertTemplate`](https://github.com/digital-asset/daml/blob/main/compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs#L922-L923) simply fails with an `unhandled` error. I think it would make more sense to follow the same approach for interfaces. You can do it in a separate pair of PRs, perhaps when you get to this [TODO in digital-asset/ghc - `RdrHsSyn`](https://github.com/digital-asset/ghc/blob/da-master-8.8.1/compiler/parser/RdrHsSyn.hs#L3183)

Reply:
Will do